solr/FULLSTORY-CHANGES.txt

```diff
@@ -1,6 +1,6 @@
 ================= FullStory ================
 (Some of these changes are available in public Solr, if we update to the specified version )
-* fs.ignoreconflicts (Scott Blum)
+* fs.ignoreconflicts (Scott Blum) (Generally available from SOLR 8.2)
```

Comment:
Feel free to move this line in favor of a `SOLR-13320:` line.  I find it helpful to be clear in this file about "forked changes" vs "legit backport changes"

Reply:
OK, so this should just look like 
`* support for failOnVersionConflicts=false , backported from SOLR-13220`

Reply:
I always just copied the jira title or git commit message, or the exact lines from the CHANGES.txt that you would have submitted with your git commit. e.g. 

`SOLR-13320: add a param failOnVersionConflicts=false to updates not fail when there is a version conflict (Noble Paul, Scott Blum) (SOLR 8.2)`

(Actually, it looks like you might have not updated Solr's CHANGES.txt in Apache)

Reply:
yeah , the cherry-pick missed it. it's committed now